core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -906,6 +906,9 @@ public PropServerConfiguration(
             boolean httpServerCookiesEnabled = getBoolean(properties, env, PropertyKey.HTTP_SERVER_KEEP_ALIVE, true);
             boolean httpReadOnlySecurityContext = getBoolean(properties, env, PropertyKey.HTTP_SECURITY_READONLY, false);
 
+            int httpQueryConnectionLimit = getInt(properties, env, PropertyKey.HTTP_QUERY_CONNECTION_LIMIT, -1);
```

Comment:
it would be good idea to validate use supplied values and also work out overall http connection limit from that. Otherwise it would be hard and error prone for the user to align all the limit so they don't override each other. For example, query + ilp > http limit - admin won't get in

Reply:
done, it adjusts the overall HTTP connection limit to `1.5 * (queryConnLimit + ilpConnLimit)` if it is less than that